MAUI/Polar-Charts/Axis/Types.md

```diff
@@ -0,0 +1,340 @@
+        <chart:NumericalAxis/>
+    </chart:SfPolarChart.SecondaryAxis>
+
+</chart:SfPolarChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfPolarChart chart = new SfPolarChart();
+. . .
+CategoryAxis primaryAxis = new CategoryAxis()
+{
+    Interval = 2, 
+};
+chart.PrimaryAxis.Add(primaryAxis);
+
+NumericalAxis secondaryAxis = new NumericalAxis();
+chart.SecondaryAxis.Add(secondaryAxis);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## DateTime Axis
+
+[DateTimeAxis]() is used to plot `DateTime` values. The [DateTimeAxis]() is widely used to make financial charts in places like the Stock Market, where index plotting is done every day.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfPolarChart>
+    . . .
+    <chart:SfPolarChart.PrimaryAxis>
+        <chart:DateTimeAxis/>
+    </chart:SfPolarChart.PrimaryAxis>
+
+</chart:SfPolarChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfPolarChart chart = new SfPolarChart();
+. . .
+DateTimeAxis primaryAxis = new DateTimeAxis();
+chart.PrimaryAxis.Add(primaryAxis);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![DateTimeAxis support in MAUI Chart](Axis_Images/MAUI_datetime_axis.png)
+
+### Interval
+
+In [DateTimeAxis](), intervals can be customized by using the [Interval]() and [IntervalType]() properties. For example, setting [Interval]() as 6 and [IntervalType]() as [Months]() will consider 6 months as interval.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfPolarChart>
+    ...
+    <chart:SfPolarChart.PrimaryAxis>
+        <chart:DateTimeAxis Interval="6" IntervalType="Months"/>
+    </chart:SfPolarChart.PrimaryAxis>
+    <chart:SfPolarChart.SecondaryAxis>
+        <chart:NumericalAxis/>
+    </chart:SfPolarChart.SecondaryAxis>
+
+</chart:SfPolarChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfPolarChart chart = new SfPolarChart();
+. . .
+DateTimeAxis primaryAxis = new DateTimeAxis()
+{
+    Interval = 6, 
+    IntervalType = DateTimeIntervalType.Months
+};
+chart.PrimaryAxis.Add(primaryAxis);
+
+NumericalAxis secondaryAxis = new NumericalAxis();
+chart.SecondaryAxis.Add(secondaryAxis);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Customizing the Range
+
+[Minimum]() and [Maximum]() properties behavior is same as in [NumericalAxis]() instead of setting numerical value, have to set date time values.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfPolarChart>
+    . . .
+    <chart:SfPolarChart.PrimaryAxis>
+        <chart:DateTimeAxis Minimum="2021/05/10" Maximum="2021/11/01"/>
+    </chart:SfPolarChart.PrimaryAxis>
+
+</chart:SfPolarChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfPolarChart chart = new SfPolarChart();
+. . .
+DateTimeAxis primaryAxis = new DateTimeAxis()
+{
+    Minimum = new DateTime(2021,05,10),
+    Maximum = new DateTime(2021,11,01),
+};
+chart.PrimaryAxis.Add(primaryAxis);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Inversed
+
+Axis can be inverted by using the [IsInversed]() property. The default value of this property is `False`.
```

Comment:
Axis can be inverted using the [IsInversed]() property. The default value of this property is `False`.